test/integration/Main.hs

```diff
@@ -84,62 +69,12 @@ main = do
         , "--log-config", stateDir <> "/logs/" <> nodeId <> "/config.json"
         , "--rebuild-db"
         ] (pure ())
-        NoStream
+        Inherit
```

Comment:
Please no :sweat_smile:  .. logs are already stored on disk in `/tmp/cardano-node-simple/logs`; let's not pollute the terminal with all the logs from the cluster. It gets VERY noisy.